setup.py

```diff
@@ -43,7 +41,7 @@ def main():
         packages=setuptools.find_packages(),
         python_requires=">=3.6",
         zip_safe=False,
-        install_requires=["requests", "requests-html", "iso639"],
+        install_requires=["iso639", "requests", "requests-html", "setuptools"],
```

Comment:
We need `setuptools` in `install_requires` now strictly speaking, because `pkg_resources` is kind-of peculiar: It has its own top-level package namespace and yet isn't pip-installable itself. It is available together with `setuptools` which is a third-party package.